rts/lra/coordinator/pom.xml

```diff
@@ -28,6 +28,11 @@
             <artifactId>microprofile-lra-api</artifactId>
             <version>${version.microprofile.lra.api}</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.fault-tolerance</groupId>
+            <artifactId>microprofile-fault-tolerance-api</artifactId>
+            <version>2.1.1</version>
```

Comment:
The versions would be better to be configurable with properties.

Reply:
Done